### DISCUSSION TITLE
Tiptap paid consulting?

### DISCUSSION THREAD

**Discussion by mattparlane** [ueberdosis/tiptap#5736] [General Questions] [answered]
I am building a text editor component for a custom CMS and I would like to use Tiptap, but I would really like some one-on-one help with it. A paid subscription is a bit out of reach at the moment, but I would be happy to pay someone on an hourly basis to help me build what we need.

Is there anyone out there who is looking for some paid work, and has experience with custom Tiptap development?

Thanks!

**Comment by nperez0111** [comment, marked as answer]
If you join the Tiptap Discord I'm sure you can find someone willing to help on the Jobs board.
